app/controllers/albums.js

```diff
@@ -1,18 +1,21 @@
 const logger = require('../logger');
 const axios = require('axios');
 const { Purchase } = require('../models');
-const { albumAlreadyPurchased, defaultError } = require('../errors');
+const { albumAlreadyPurchased, defaultError, dbError } = require('../errors');
 
 module.exports = {
   list(req, res, next) {
-    axios.get('https://jsonplaceholder.typicode.com/albums').then(function(response) {
-      res.status(200).json({
-        albums: response.data.map(album => {
-          delete album.userId;
-          return album;
-        })
-      });
-    });
+    axios
+      .get('https://jsonplaceholder.typicode.com/albums')
+      .then(function(response) {
+        res.status(200).json({
+          albums: response.data.map(album => {
+            delete album.userId;
+            return album;
+          })
+        });
+      })
+      .catch(err => next(defaultError('External API error.')));
```

Comment:
maybe we can add the original error message from `err` also

Reply:
Fixed